src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -11,65 +11,42 @@ redirects:
   - /docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-usage-metrics-performance
 ---
 
-Mobile monitoring provides a **Version trends** report with metrics to compare usage, adoption, and performance across the most recent versions of your mobile app. This helps mobile app development managers and their teams to compare up to five production versions from a single page and to analyze the impact of improvements, fixes, and degradations for each.
-
-You can view version trend data in the mobile monitoring UI. You can also enable [email notifications](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/review-version-trends-email-reports).
+You can view version trend data in the mobile monitoring UI. Here you can compare performance across the most recent versions of your mobile app using the version trends report. On this single page you have access to the key data points you need to analyze the impact of improvements, fixes, and degradations for each version. This report provides metrics to compare usage, adoption, and key technical indicators.
```

Comment:
How are "key technical indicators" different from the other indicators? Is it possible to get more specific there?